docs/UserGuide.md

```diff
@@ -156,6 +156,19 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd student in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st student in the results of the `find` command.
 
+### Receiving payment from a student : `pay`
+
+Updates the amount of tuition fee paid by the specified student after a lesson.
+
+Format: `pay INDEX h/HOURS_PAID`
```

Comment:
I think it is great that ur prefix is very short. However, in this case, do you think it'll be abit clearer if you use hr/ instead?

Reply:
I think you are right. I have changed the prefix to make it clearer.